mybatis-spring-boot-autoconfigure/src/main/java/org/mybatis/spring/boot/autoconfigure/MybatisAutoConfiguration.java

```diff
@@ -121,10 +126,19 @@ public SqlSessionFactory sqlSessionFactory(DataSource dataSource) throws Excepti
 			factory.setTypeHandlersPackage(this.properties.getTypeHandlersPackage());
 			factory.setMapperLocations(this.properties.resolveMapperLocations());
 		}
-
 		return factory.getObject();
 	}
 
+	/**
+	 * @since 1.1.0
+	 */
+	@Bean
+	@ConditionalOnMissingBean
+	@ConfigurationProperties(prefix = MybatisProperties.MYBATIS_PREFIX + ".configuration")
+	public org.apache.ibatis.session.Configuration myBatisConfiguration() {
+		return new org.apache.ibatis.session.Configuration();
```

Comment:
I doesn't have reasonable reason.
Therefor, i moved to `MybatisProperties` via 5670187f7b483125be4f06e145cf806ef36143e0.